ouroboros-consensus-test/src/Test/Util/Serialisation/Roundtrip.hs

```diff
@@ -383,6 +421,15 @@ roundtrip_SerialiseNodeToClient ccfg =
     , rt (Proxy @(GenTx blk))                 "GenTx"
     , rt (Proxy @(ApplyTxErr blk))            "ApplyTxErr"
     , rt (Proxy @(SomeSecond BlockQuery blk)) "BlockQuery"
+    -- Note: Ideally we'd just use `rt` to test Ledger config, but that would
+    -- require an Eq and Show instance for all ledger config types which we'd
+    -- like to avoid.
+    , testProperty ("roundtrip (comparing encoding) LedgerConfig") $ forAllShow
```

Comment:
Would you use the `Blind` QC modifier instead of `forAllShow`? `forAllShow` does now shrinking :/

Reply:
Oh yes, good idea!